octoprint_CameraSettings/static/js/CameraSettings.js

```diff
@@ -91,23 +91,16 @@ $(function() {
             auto_white_balance_speed: { use: ko.observable(false), value: ko.observable(undefined), min: ko.observable(0), max: ko.observable(100), step: ko.observable(1) },
             auto_white_balance_delay: { use: ko.observable(false), value: ko.observable(undefined), min: ko.observable(0), max: ko.observable(100), step: ko.observable(1) },
 
-            save_user_settings: { use: ko.observable(false) },
-            restore_user_settings: { use: ko.observable(false) },
-            restore_factory_settings: { use: ko.observable(false) },
-            pan_reset: { use: ko.observable(false) },
-            tilt_reset: { use: ko.observable(false) },
```

Comment:
If these are deleted then users with cameras that report these controls will have errors.